tldr/cross_file_calls/core.py

```
@@ -0,0 +1,305 @@
+    @property
+    def edges(self) -> set[tuple[str, str, str, str]]:
+        """Return all edges as a set of tuples."""
+        return self._edges
+
+    def __contains__(self, edge: tuple[str, str, str, str]) -> bool:
+        """Check if an edge exists in the graph."""
+        return edge in self._edges
+
+
+# Parser factory functions
+def _get_ts_parser():
```

Ruff (0.14.11)

Missing return type annotation for private function `_get_ts_parser`

(ANN202)